<p>
  Each element should be on a line to itself.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
&ltparent&gt&ltchild /&gt&lt/parent&gt &lt!-- Noncompliant --&gt
</pre>

<h2>Compliant Solution</h2>
<pre>
&ltparent&gt
  &ltchild /&gt
&lt/parent&gt
</pre>

